src/scripts/content/google-calendar.js

```diff
@@ -20,7 +20,8 @@ togglbutton.render('div[data-chips-dialog="true"]', { observe: true }, elem => {
 
   const link = togglbutton.createTimerLink({
     className: 'google-calendar-modern',
-    description: getDescription
+    description: getDescription,
+    buttonType: 'minimal'
```

Comment:
we shouldn't change the button to minimal. It's unfortunate that some graphic backgrounds will clash with the text, but we want to keep the full button for regular events.

Reply:
OK!
I changed it.

![image](https://user-images.githubusercontent.com/13980441/50748037-9bc1f300-127a-11e9-8b87-a6a4cc8487b1.png)